packages/next-app/src/app/metadata/[network]/pool/[poolId]/(components)/MetadataAttributesTable/PredefinedMetadataModal.tsx

```diff
@@ -0,0 +1,196 @@
+import { TypenameEnum } from "@balancer-pool-metadata/schema";
+import * as Accordion from "@radix-ui/react-accordion";
+import * as Dialog from "@radix-ui/react-dialog";
+import { useState } from "react";
+
+import { Button } from "#/components";
+import {
+  AccordionContent,
+  AccordionItem,
+  AccordionTrigger,
+} from "#/components/Accordion";
+import { Checkbox } from "#/components/Checkbox";
+import { usePoolMetadata } from "#/contexts/PoolMetadataContext";
+import { toSlug } from "#/utils/formatStringCase";
+
+interface Itemplate {
+  name: string;
+  options: {
+    name: string;
+    type: (typeof TypenameEnum.enum)[keyof typeof TypenameEnum.enum];
+  }[];
+}
+
+export const ATTRIBUTES_TEMPLATE: Itemplate[] = [
+  {
+    name: "Basic Data",
+    options: [
+      { name: "Name", type: "text" },
+      { name: "Symbol", type: "text" },
+      { name: "Creator", type: "text" },
+      { name: "Date creation", type: "date" },
+      { name: "Website", type: "url" },
+    ],
+  },
+  {
+    name: "Managed Pools",
+    options: [
+      { name: "Report", type: "text" },
+      { name: "History", type: "text" },
+      { name: "Changelog", type: "text" },
+    ],
+  },
+  {
+    name: "Boosted Pools",
+    options: [
+      { name: "Summary", type: "text" },
+      { name: "Changelog", type: "text" },
+      { name: "Risks", type: "text" },
+    ],
+  },
+];
```

Comment:
would avoid defining new names for things that already have a naming convention
```suggestion
export const ATTRIBUTES_TEMPLATE: Itemplate[] = [
  {
    name: "Basic Data",
    options: [
      { key: "Name", typename: "text" },
      { key: "Symbol", typename: "text" },
      { key: "Creator", typename: "text" },
      { key: "Date creation", typename: "date" },
      { key: "Website", typename: "url" },
    ],
  },
  {
    name: "Managed Pools",
    options: [
      { key: "Report", typename: "text" },
      { key: "History", typename: "text" },
      { key: "Changelog", typename: "text" },
    ],
  },
  {
    name: "Boosted Pools",
    options: [
      { key: "Summary", typename: "text" },
      { key: "Changelog", typename: "text" },
      { key: "Risks", typename: "text" },
    ],
  },
];
```

Reply:
That makes sense